### ISSUE TITLE
`warning: unused namespace ...` and `schema.core/defn`

### ISSUE THREAD

**Issue by mynomoto** [borkdude/clj-kondo#163]
There are some false positives where the namespace is only used to add schemas to functions defined with `schema.core/defn`. 
```clojure
(ns clj-kondo-test.schema
 (:require
    [another-ns.schema :as s]
    [schema.core :as sc]))

(sc/defn my-identity :- s/MyType
  [amount :- s/AnotherType]
  amount)
```
Gives the warning `warning: unused namespace another-ns.schema`

**Comment by borkdude** [comment]
Thank you.

**Comment by borkdude** [comment]
Fixed with c8837c94349f4c6b633bdfadef968b63c894d72b
